tests/lib/hardware/hardware_base.py

```diff
@@ -53,6 +53,7 @@ def __init__(self):
         self._working_dir: str = tempfile.mkdtemp(
             prefix="%s%s_" % (config.CLUSTER_PREFIX, self.hardware_uuid))
 
+        logger.info(f"hardware {self.hardware_uuid}: Using {self.working_dir}")
```

Comment:
Should be self._working_dir ?

Reply:
no. it's a read only property

Reply:
Ah, that's right (was just seeing the previous line).